picard/tagger.py

```diff
@@ -274,6 +274,7 @@ def exit(self):
         self.stopping = True
         self._acoustid.done()
         self.thread_pool.waitForDone()
+        self.save_thread_pool.waitForDone()
```

Comment:
I assume that if you have 20 files queued up it doesn't start any new saves from the queue before quitting.